test/built-ins/BigInt/prototype/valueOf/this-value-err.js

```diff
@@ -0,0 +1,17 @@
+// Copyright (C) 2017 Robin Templeton. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-bigint.prototype.valueof
+description: BigInt.prototype.valueOf this value coercion
+info: >
+  BigInt.prototype.valueOf ( )
+
+  Return ? thisBigIntValue(this value).
+includes: [typeCoercion.js]
+features: [BigInt, arrow-function, Symbol, Symbol.toPrimitive]
```

Comment:
this seems to have more tagged features than necessary. Is Symbol.toPrimitive ever used?